services/apps/src/apps/routes/schemas/groups.clj

```diff
@@ -58,15 +58,6 @@
 (s/defschema GroupMembersUpdate
   {:members (describe [NonBlankString] "The new list of member subject IDs.")})
 
-(s/defschema GroupMemberSubjectUpdateResponse
-  {:success
-   (describe Boolean "True if the user was added successfully")
-
-   :subject_id
-   (describe NonBlankString "The subject ID.")
-
-   (s/optional-key :subject_name)
-   (describe NonBlankString "The subject name.")})
-
 (s/defschema GroupMembersUpdateResponse
-  {:results (describe [GroupMemberSubjectUpdateResponse] "The list of membership update results.")})
```

Comment:
This is minor, but it might be nice to implement this schema as an addition to the existing `GroupMembers` schema rather than something separate (i.e. something like `(assoc GroupMembers :failures (...))`). I tend to think this is better since it conveys the intent better -- it's a list of the new group members with an extra field added, plus it makes it a bit more DRY.

(but, definitely not a bug or something that has to be fixed, just food for thought)